src/main/java/com/powsybl/openloadflow/dc/DcLoadFlowEngine.java

```diff
@@ -139,7 +137,7 @@ private static Pair<Boolean, double[]> run(LfNetwork network, DcLoadFlowParamete
             distributeSlack(remainingBuses, parameters.getBalanceType());
         }
 
-        var targetVectorArray = targetVector.getArray();
+        var targetVectorArray = targetVector.getArray().clone();
```

Comment:
@annetill this fix is the good one for me, because indeed we need to work on a copy of target vector, because first it could be hacked with disabled element and also because it is updated by the LU decomposition result and so on if we want to keep targets for later use we need to copy.

Reply:
maybe just add an explanation

Reply:
Done